extensions/wikia/ExactTargetUpdates/client/builders/UpdateRequestBuilder.class.php

```diff
@@ -7,10 +7,35 @@
 class UpdateRequestBuilder extends BaseRequestBuilder {
 	const SAVE_OPTION_TYPE = 'SaveOption';
 
+	private $categories = [];
+	private $cityId;
+	private $cityData;
 	private $userData;
 	private $edits;
 
-	private static $supportedTypes = [ self::PROPERTIES_TYPE, self::EDITS_TYPE, self::USER_TYPE ];
+	private static $supportedTypes = [
+		self::PROPERTIES_TYPE,
+		self::EDITS_TYPE,
+		self::USER_TYPE,
+		self::WIKI_TYPE,
+		self::WIKI_CAT_TYPE
+	];
+
+	public function withCategories( $categories ) {
```

Comment:
Back in the past, wikis were referred to as cities. I think it would be a good idea to stick to a single naming convention and use `withWikiId()`, `withWikiData()`, `$this->wikiId` so the only usage of `city` is in constants representing MySQL column names.